README.md

```diff
@@ -1,2 +1,69 @@
+
+### Deliverable
+One console application that represents the application as described. You should provide access to an online repository, based on this one that contains all source code for this application, along with complete instructions on how to build your application locally. If any tests or libraries were written, please provide code for all and full instructions on how to run the build and tests.
+
+### Acceptance criteria
+1. The SLMM never goes outside of the dimensions of the garden as supplied during startup
+2. The output of the SLMM after it finished each action must be in the format: “{Time} – {Action taken place} – {current location of SLMM}”
+3. The commands to be accepted in this version are:
+
+
+Step 2 – senior & lead server-side positions
+-------------------------------------
+Please complete the above first. After you finished the above, you are to enhance it with the following:
+1. The application should now be changed to accept input via Http. You are advised to use ASP.Net Web API, but you are free to use any web framework of your choice. The output should still be in the console.
+2. Since now this is an ASP.Net Web API, the size of the lawn should be passed in through an action method specifically for this purpose.
+3. All commands should be done asynchronously and all action methods should just return a correlation id.
+4. The commands should be done on a FIFO manner.
+5. There should be two applications, one that runs in SLMM and one that represents the client and will be used by administrator(s) to control the SLMM. See below for accepted commands and exposed action methods.
+
+### Commands accepted by SLMM client
+1. TL -> Turn Left
+2. TR -> Turn Right
+3. MF -> Move Forward
+4. ML -> Mow Lawn
+
+### Action methods exposed by SLMM
+1. /lawn Method: POST. Payload: {"StartX": int, "StartY": int, "SizeX": int, "SizeY": int}
+2. /location Method: PUT. Payload: {"Units": int}
+3. /location Method: GET. Reply: {"X": int, "Y": int} -> returns the current location of the SLMM
+4. /mower Method: PUT Payload {"On": boolean}
+
+### Deliverables
+First provide the deliverables for Step 1 independantly. Then provide one console application that represents the application that is run in the SLMM and one console application that accepts command input from console and communicates it to the SLMM application via Http along with all supporting libraries. Again, you should provide access to an online repository that hosts all code for all applications, libraries and test projects (we recommend this to be in another branch than the one for step 1). The readme should have all necessary information on how to build, run and test the solution.
+
+### Acceptance criteria
+1. The SLMM never goes outside of the dimensions of the garden as supplied during startup
+2. In the SLMM application, it is expected that action methods will log when they are called, start of operations and end of operations should be logged as well.
```

Comment:
I am trying to write in an active, current verse. So I think maybe "should log when they get called"?